src/components/tab-nav/tab-nav.tsx

```diff
@@ -233,7 +241,7 @@ export class TabNav {
     }
   }
 
-  @Listen("calciteTabChange", { target: "body" }) globalTabChangeHandler(
+  @Listen("calciteInternalTabChange", { target: "body" }) globalInternalTabChangeHandler(
```

Comment:
☝️  Since this function is listening for an internal event, should we stop propagation and/or prevent default on this one like we do above with the `calciteInternalTabsActivate` event?